core/domain/exp_jobs_one_off.py

```diff
@@ -490,3 +493,108 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class VoiceoverDurationSecondsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-Off Job to set every voiceover's duration to duration_secs with
+    the correct value.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_fetchers.get_exploration_by_id(item.id)
+        try:
+            exploration.validate()
+        except Exception as e:
+            logging.error(
+                'Exploration %s failed non-strict validation: %s' %
+                (item.id, e))
+            return
+        if (item.states_schema_version <
+                AUDIO_DURATION_SECS_MIN_STATE_SCHEMA_VERSION):
+            yield ('State schema version is not the '
+                   'minimum version expected', item.id)
```

Comment:
Maybe include, in the second part of the yield statement, the value of item.states_schema_version.

Reply:
ok

Reply:
done